crates/orderbook/openapi.yml

```diff
@@ -604,19 +604,20 @@ components:
         Provides the additional data for ethflow orders
       type: object
       properties:
-        isRefunded:
```

Comment:
I added it back in [f68265a](https://github.com/cowprotocol/services/pull/970/commits/f68265afcb08abc31ef118dae1fbc27a86e10d86) again. Technically the field is redundant and only needed during serialization so I'll try to look how this could get implemented with `serde`. But to not delay possible testing I already implemented the basic solution.